vmdb/app/helpers/api_helper/renderer.rb

```diff
@@ -198,6 +198,7 @@ def collection_search(is_subcollection, type, klass)
         :userid         => @auth_user,
         :results_format => :objects
       }
+      options[:order] = sort_options              if sort_options.present? && res.respond_to?(:reorder)
```

Comment:
You have this same conditional just 7 lines up.  Can they be mixed together somehow?